.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "jest": true
+
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+
+  "globals": {
+    "__dirname" : false
+  }
+}
```

Comment:
Configure this file, same as the one from [MICROVERSE's config](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) file.
You will surely get stickler errors to correct  😊